src/4626/MoonwellERC4626.sol

```diff
@@ -8,13 +8,13 @@ import {ERC20} from "solmate/tokens/ERC20.sol";
 import {MErc20} from "@protocol/MErc20.sol";
 import {MToken} from "@protocol/MToken.sol";
 import {LibCompound} from "@protocol/4626/LibCompound.sol";
-import {Comptroller as IComptroller} from "@protocol/Comptroller.sol";
+import {Comptroller as IMoontroller} from "@protocol/Comptroller.sol";
 import {MultiRewardDistributor, MultiRewardDistributorCommon} from "@protocol/rewards/MultiRewardDistributor.sol";
 
-/// @title CompoundERC4626
+/// @title MoonwellERC4626
 /// @author zefram.eth
 /// @notice ERC4626 wrapper for Moonwell Finance
-contract CompoundERC4626 is ERC4626 {
+contract MoonwellERC4626 is ERC4626 {
```

Comment:
There are still compound naming references in error `CompoundERC4626__CompoundError`, comments and variable name like `comptroller`.

Reply:
yes right. but still there are some names in this file that correspond to compound like the error `CompoundERC4626__CompoundError` and `IMoontroller public immutable comptroller;`